hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/realtime/HoodieRealtimeInputFormat.java

```diff
@@ -19,7 +19,6 @@
 package com.uber.hoodie.hadoop.realtime;
 
 import com.google.common.base.Preconditions;
-
```

Comment:
What has changed in these lines? Can you reset this file if nothing has changed.